ecl/agentexec/agentexec.hpp

```diff
@@ -17,30 +17,5 @@
 #ifndef AGENTEXEC_SERVER_HPP
```

Comment:
Should the whole file be deleted? (or entry restored to CMakeLists.txt)

Reply:
Yes it should